blueoil/converter/templates/Makefile

```diff
@@ -108,139 +87,108 @@ clean:
-ar_x86_avx:       LDFLAGS += -rcs
-ar_x86_avx:       NAME = x86_avx
-
-ar_aarch64:       AR = aarch64-linux-gnu-ar
-ar_aarch64:       CXX = aarch64-linux-gnu-g++
-ar_aarch64:       FLAGS += $(INCLUDES) -O3 -std=c++14 -fPIC -fvisibility=hidden -DUSE_NEON -pthread -g
-ar_aarch64:       LDFLAGS += -rcs
-ar_aarch64:       NAME = aarch64
-
-ar_arm:           AR = arm-linux-gnueabihf-ar
-ar_arm:           CXX = arm-linux-gnueabihf-g++
-ar_arm:           FLAGS += $(INCLUDES) -O3 -std=c++14 -fPIC -DUSE_NEON -DAARCH32 -mcpu=cortex-a9 -mfpu=neon -mthumb -fvisibility=hidden -pthread -g -fopenmp
-ar_arm:           LDFLAGS += -rcs
-ar_arm:           NAME = arm
-
-ar_fpga:          AR = arm-linux-gnueabihf-ar
-ar_fpga:          CXX = arm-linux-gnueabihf-g++
-ar_fpga:          FLAGS += $(INCLUDES) -O3 -std=c++14 -fPIC -DUSE_NEON -DRUN_ON_FPGA -DAARCH32 -mcpu=cortex-a9 -mfpu=neon -mthumb -fvisibility=hidden -pthread -g -fopenmp
-ar_fpga:          LDFLAGS += -rcs
-ar_fpga:          NAME = fpga
-
-
-$(TARGETS_ARM): $(OBJ) $(LIB_ARM_OBJ)
-	$(CXX) $(FLAGS) $(OBJ) $(LIB_ARM_OBJ) -o $@.elf $(CXXFLAGS) -pthread -ldl
-
-$(TARGETS_FPGA): $(OBJ) $(LIB_FPGA_OBJ)
-	$(CXX) $(FLAGS) $(OBJ) $(LIB_FPGA_OBJ) -o $@.elf $(CXXFLAGS) -pthread -ldl
-
-$(TARGETS_AARCH64): $(OBJ) $(LIB_AARCH64_OBJ)
-	$(CXX) $(FLAGS) $(OBJ) $(LIB_AARCH64_OBJ) -o $@.elf $(CXXFLAGS) -pthread -ldl
-
-$(TARGETS_X86): $(OBJ) $(LIB_X86_OBJ)
-	$(CXX) $(FLAGS) $(OBJ) $(LIB_X86_OBJ) -o $@.elf $(CXXFLAGS) -pthread -ldl
-
-$(TARGETS_X86_AVX): $(OBJ) $(LIB_X86_AVX_OBJ)
-	$(CXX) $(FLAGS) $(OBJ) $(LIB_X86_AVX_OBJ) -o $@.elf $(CXXFLAGS) -pthread -ldl
-
-$(LIBS_X86): $(LIB_OBJ) $(LIB_X86_OBJ)
-	$(CXX) $(FLAGS) $(LIB_OBJ) $(LIB_X86_OBJ) -o libdlk_$(NAME).so $(CXXFLAGS) -shared -pthread -ldl
-
-$(LIBS_X86_AVX): $(LIB_OBJ) $(LIB_X86_AVX_OBJ)
-	$(CXX) $(FLAGS) $(LIB_OBJ) $(LIB_X86_AVX_OBJ) -o libdlk_$(NAME).so $(CXXFLAGS) -shared -pthread -ldl
-
-$(LIBS_AARCH64): $(LIB_OBJ) $(LIB_AARCH64_OBJ)
-	$(CXX) $(FLAGS) $(LIB_OBJ) $(LIB_AARCH64_OBJ) -o libdlk_$(NAME).so $(CXXFLAGS) -shared -pthread -ldl
-
-$(LIBS_ARM): $(LIB_OBJ) $(LIB_ARM_OBJ)
-	$(CXX) $(FLAGS) $(LIB_OBJ) $(LIB_ARM_OBJ) -o libdlk_$(NAME).so $(CXXFLAGS) -shared -pthread -ldl
-
-$(LIBS_FPGA): $(LIB_OBJ) $(LIB_FPGA_OBJ)
-	$(CXX) $(FLAGS) $(LIB_OBJ) $(LIB_FPGA_OBJ) -o libdlk_$(NAME).so $(CXXFLAGS) -shared -pthread -ldl
-
-$(ARS_X86): $(LIB_OBJ) $(LIB_X86_OBJ)
-	$(AR) $(LDFLAGS) libdlk_$(NAME).a $(LIB_OBJ) $(LIB_X86_OBJ)
-
-$(ARS_X86_AVX): $(LIB_OBJ) $(LIB_X86_AVX_OBJ)
-	$(AR) $(LDFLAGS) libdlk_$(NAME).a $(LIB_OBJ) $(LIB_X86_AVX_OBJ)
-
-$(ARS_AARCH64): $(LIB_OBJ) $(LIB_AARCH64_OBJ)
-	$(AR) $(LDFLAGS) libdlk_$(NAME).a $(LIB_OBJ) $(LIB_AARCH64_OBJ)
-
-$(ARS_ARM): $(LIB_OBJ) $(LIB_ARM_OBJ)
-	$(AR) $(LDFLAGS) libdlk_$(NAME).a $(LIB_OBJ) $(LIB_ARM_OBJ)
-
-$(ARS_FPGA): $(LIB_OBJ) $(LIB_FPGA_OBJ)
-	$(AR) $(LDFLAGS) libdlk_$(NAME).a $(LIB_OBJ) $(LIB_FPGA_OBJ)
+	-$(RM) $(ALL_OBJ)
+	-$(RM) $(ALL_DEP)
+
+
+# Base Settings
+FLAGS := -I./include -O3 -std=c++14 -g -MMD -MP
```

Comment:
`CXXFLAGS` is commonly used for C++ compile flags.

Reply:
@primenumber 
The FLAGS variable has been replaced with CXXFLAGS instead